packages/financial-templates-lib/package.json

```diff
@@ -23,6 +23,7 @@
     "winston-transport": "^4.3.0"
   },
   "devDependencies": {
+    "bignumber.js": "^9.0.1",
```

Comment:
Is this an accidental import?

Reply:
Nope, bignumber.js was included as a dev dep so we could use it in the tests (I pulled a function from uniswap's tests where they use it to convert prices). Would you prefer that we drop it in favor of just using web3's native BN?

Reply:
nope just double checking

Reply:
If we _do_ keep it then you can change the `decodePriceSqrt` computation :)